t/06-blacklist.t

```diff
@@ -14,7 +17,7 @@ ok($contents, "Got contents from original blacklist");
 
 diag `perl $FindBin::Bin/../tools/makeblacklist.pl`;
 
-open $blist, '<', "$sqldir/BLACKLIST";
+open $blist, '<', "$tempdir/BLACKLIST";
```

Comment:
The original is read at line 12. This is the new copy and yes, I want to read it from the temporary copy I just generated.